2_phase_minting_schedule/main.tex

```diff
@@ -70,74 +71,103 @@ \section{Empirical analysis of existing networks}
 \end{equation}
 
 We cancel out $S_0$ and solve the second integral.
 
 \begin{equation}
-    3.885 = 1 + 5 \cdot  I_{static} + I_{static} \cdot \left[\frac{-2^{(1-\frac{t}{2})}}{\ln{2}}\right]_0^\infty
+    3.885 = 1 + 5 \cdot  I_{stable} + I_{stable} \cdot \left[\frac{-2^{(1-\frac{t}{2})}}{\ln{2}}\right]_0^\infty
 \end{equation}
 
-We are now able to find an expression for $I_{static}$.
+We are now able to find an expression for $I_{stable}$. Once again note that this variable is expressed as a percentage of the genesis supply, $S_0$.
 
 \begin{equation}
-    2.885 = I_{static} \cdot (5 + \frac{2}{\ln{2}})
+    2.885 = I_{stable} \cdot (5 + \frac{2}{\ln{2}})
 \end{equation}
 
 \begin{equation}
-   I_{static} = \frac{2.885}{7.885} = 36.6\%
+   I_{stable} = \frac{2.885}{7.885} = 36.6\%
 \end{equation}
 
-This is equivalent to a daily inflation rate of 0.10\% in the first five years, equivalent to the first 1826 periods.
-\\\\
-Note that the inflation rate of 0.10\% is applied, in each of the first 1825 periods, \textbf{to the genesis supply} ($S_0$ = 1 billion), not, as some may assume, to the variable supply ($S$ $>$ 1 billion). In other words, the supply grows by a constant number of tokens every period, which implies that the `real' percentage growth rate of the supply actually decreases period-to-period. 
+We use $I_{stable}$ to find the number of tokens minted per year for the first 5 years.
+
+\begin{equation}
+   I_{stable} \cdot S_0 = 365,915,960\:\:new \:tokens \:per \:year
+\end{equation}
+
+This is equivalent to 1,002,509 newly minted tokens released into the circulating supply, every period (day) for the first 1825 periods. After this, the issuance decays on a period-to-period basis. For example, 1,001,558 tokens are minted on day 1827, which is 951 fewer than the previous day.
 \\\\
-To avoid confusing the inflation rate with the real growth rate of the supply, we hereafter suggest the term \textit{minting schedule} to describe the growth of the circulating supply.
+In this token minting schedule, we can work out that approximately 1.83 billion tokens will be minted in the first phase, followed by a further 1.056 billion between the commencement of phase two and t = $\infty$.
+
+\subsection{Variable stake lock-up duration}
+
+In the NuCypher protocol, the number of tokens distributed to node operators in a given period depends on the stage in the minting schedule we are at and the size of their stake, but is also a function of the remaining duration their stake is locked up for (in fact, the duration of each of the `sub-stakes' they control). Any sub-stake locked up for a short duration receives less than the maximum subsidy, thereby slowing the expansion of the entire circulating supply – tokens are only minted if they are to be distributed to node operators. The number of tokens minted daily for a given sub-stake depends on the coefficient $\kappa$:
```

Comment:
"depends on the [current] stage in the minting schedule and..."
"also a function of the remaining [locked duration of their stake] (in fact..."
"Any sub-stake [locked] for a ..."

```suggestion
In the NuCypher protocol, the number of tokens distributed to node operators in a given period depends on the current stage in the minting schedule and the size of their stake, but is also a function of the remaining locked duration of their stake (in fact, the duration of each of the `sub-stakes' they control). Any sub-stake locked for a short duration receives less than the maximum subsidy, thereby slowing the expansion of the entire circulating supply – tokens are only minted if they are to be distributed to node operators. The number of tokens minted daily for a given sub-stake depends on the coefficient $\kappa$:
```